ykrt/src/mt.rs

```diff
@@ -305,6 +305,10 @@ impl MTThread {
                             .unwrap();
                         let tir_trace = TirTrace::new(&*SIR, &*sir_trace).unwrap();
                         let ct = TraceCompiler::<I>::compile(tir_trace);
+                        // Now that we've got a compile trace, store it within the tag. However,
```

Comment:
"store it within the *tag*".

Reading the code, it seems that the "pack" is the name we gave the whole `u64` and the "tag" is the two bits reserved to identify the rest of the pack.

"compile" -> "compile*d*".

I think a small rewording would make it easier to read. How about:

> Now that we have a compiled trace, we need to put a pointer to it inside the corresponding Location's pack. To inhibit ExecutableBuffer's Drop, we move it into a box and forget it: the pointer in the pack is now the sole handle on the trace.

Reply:
Fixed in 4152f09.